server/services/appointmentService.ts

```diff
@@ -67,7 +67,7 @@ export default class AppointmentService {
               : apt.category.description,
             dpsLocationId: apt.internalLocation?.dpsLocationId,
             locationDescription: apt.internalLocation?.description || (apt.inCell ? 'In cell' : undefined),
-            status: apt.isCancelled ? 'CANCELLED' : 'ACTIVE',
+            status: apt.isCancelled || apt.isDeleted ? 'CANCELLED' : 'ACTIVE',
```

Comment:
Needed to generate the A&A types to pick up the isDeleted.  Fortunately for us this has very recently been made visible (17 Sept 2025).

Reply:
I should add this (`isDeleted`) is available in prod, I did ask/check.